test/templates/button-group/index.njk

```diff
@@ -1,129 +1,44 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
-{% from "../button/component.njk" import button %}
 {% from "./component.njk" import button_group %}
 
+
 {% block content %}
 <style media="screen">
   .fd-button-group {
     margin-bottom: 10px;
   }
 </style>
-<h2>default</h2>
-
-
-    {% for size in ["","compact"] %}
-      {%- set modifier = { block: [size] } %}
-      {%- if size == "default" %}
-        {%- set modifier = { block: [] } %}
-      {%- endif %}
-<h2> {{size}}</h2>
-
-{% set example %}
-
-  {% call button_group() -%}
-    {{-  button(
-      { icon: 'survey' },
-        modifier={ block: [size] },
-        aria={ label: 'Survey' })
-      }}
-    {{-  button(
-        { icon: 'pie-chart' },
-          modifier={ block: [size] },
-          aria={ label: 'Chart', selected: true })
-        }}
-    {{-  button(
-          { icon: 'pool' },
-            modifier={ block: [size] },
-            aria={ label: 'Pool' })
-          }}
-  {%- endcall %}
-
-  {% call button_group() -%}
-    {{-  button(
-      { label: 'First' },
-        modifier={ block: [size] },
-        aria={ selected: true })
-      }}
-    {{-  button(
-        { label: 'Second' },
-          modifier={ block: [size] },
-          aria={ })
-        }}
-    {{-  button(
-          { label: 'Third' },
-            modifier={ block: [size] },
-            aria={ })
-          }}
-  {%- endcall %}
 
+<style>
+  .btn-playground {
```

Comment:
@greg-a-smith Can we move this to layout.css or some other shared css file. I think we have a lot of components coming up that need a placeholder button.